### ISSUE TITLE
The `aria-current` docs aren't clear about how to handle a "current" nav-item that isn't a link-to-current-page.

### ISSUE THREAD

**Issue by MichaelAllenWarner** [w3c/aria-practices#2583]
# Describe your concern

It's clear enough that if a link in a set of nav-links goes to the current page, then it should get `aria-current="page"`. However, there are other situations where an item in a navigational structure is somehow "current" (and potentially styled accordingly) but is not a link-to-current-page, and here it's not clear from the docs what should be done.

For example, sometimes a main-nav consists only of top-level links, and the "current" style (if present) then indicates merely that the user is currently on some page within the corresponding section of the site. In this case, `aria-current="page"` presumably wouldn't be appropriate. Should one use `aria-current="location"`? Or `aria-current="true"`? Or no `aria-current` attribute at all?

Another example would be a multi-level navigational structure, where the current-page-link is nested at the bottom of a "tree." Here, higher-level elements in the current-link's "branch" are _also_ "current" in a way that someone using a screen-reader might appreciate being made aware of. What value (if any) should be used for `aria-current` on those higher-level "current" items?

I think that fleshing out the `aria-current` documentation a bit more could help clarify these kinds of situations for developers. 

(From what I'm seeing in the sidebar of [this W3 page](https://www.w3.org/WAI/videos/standards-and-benefits/), I'd guess that `aria-current="location"` is the appropriate attribute to use for the scenarios I described above, but I'm not sure.)

# Link to the version of the specification or documentation you were looking at at.

Link to documentation: https://w3c.github.io/aria/#aria-current

Does the issue exists in [the editors draft](https://w3c.github.io/aria/) (the editors draft is the most recent draft of the specification)?

Yes

**Comment by mcking65** [comment]
@MichaelAllenWarner 

The TF recommendation is to use current=true on ancestor pages represented in site navigation structures, e.g. a top-level menu that includes a link to an ancestor page that is styled as current.

**Comment by mcking65** [comment]
One open question on this is where the TF recommendation on this question should be communicated in the APG.
